Functions/Mock.ps1

```diff
@@ -1544,3 +1547,75 @@ function Remove-MockFunctionsAndAliases {
         & $script:SafeCommands['Remove-Item'] "function:/$($bootstrapFunction.Name)"
     }
 }
+
+
+function Repair-ConflictingParameters {
+    [CmdletBinding()]
+    [OutputType([System.Management.Automation.CommandMetadata])]
+    param(
+        [Parameter(Mandatory)]
+        [System.Management.Automation.CommandMetadata]
+        $Metadata
+    )
+
+    $paramMetadatas = @()
+    $paramMetadatas += $Metadata.Parameters.Values
+
+    foreach ($paramMetadata in $paramMetadatas) {
+        if ($paramMetadata.IsDynamic) {
+            continue
+        }
+
+        $conflictingParams = Get-ConflictingParameterNames
+        if ($conflictingParams -contains $paramMetadata.Name) {
+            $paramName = $paramMetadata.Name
+            $newName = "$(Get-ParameterPrefixForConflictingParameters)$paramName"
+            $paramMetadata.Name = $newName
+            $paramMetadata.Aliases.Add($paramName)
+
+            $Metadata.Parameters.Remove($paramName) | Out-Null
```

Comment:
use `$null =` instead, `Out-Null` is slow, and user might have mocked it, so you'd need to use out-null from the safe commands table which adds even more overhead.

Reply:
Done